docs/upgrading.md

```diff
@@ -13,57 +13,6 @@ Upgrading your Expo project to a new version of React Native requires updating t
 
 Because typical React Native projects are essentially made up of an Android project, an iOS project, and a JavaScript project, upgrading can be rather tricky. There's currently two ways for upgrading your React Native project: by using [React Native CLI](https://github.com/react-native-community/cli) or manually with [Upgrade Helper](https://react-native-community.github.io/upgrade-helper/).
```

Comment:
Here we've mentioned there are two ways, so you should update this and the wording around it accordingly, at the end the flow of this doc should be according to the headings and suheadings👍

Reply:
Good catch!